rustfmt.toml

```diff
@@ -0,0 +1,64 @@
+max_width = 100
```

Comment:
Please remove all default values so it's clear what the non-default values are.

Reply:
`rustfmt.toml` was generated by `rustfmt --print-config default rustfmt.toml`. There're two other options, `current` and `minimal`. Do you mean I should go for `minimal` or keep the default and remove some of them (which ones excatly?) ?

Reply:
I think we should start with an empty file and only set options we want to customize.  We probably want `newline_style=Unix` so we don't end up with different files having different line endings and `edition="2018"` in case someone invokes rustfmt directly without going through cargo fmt.

Reply:
Sounds good! I also kept major `version="One"` for consistency across.